.githooks/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Redirect output to stderr.
+exec 1>&2
+make black
+git diff --cached --name-status | cut -f 2 | xargs -I % git add -u %
```

Comment:
Shouldn't we therapist? Just to align the tooling with Kinto/kinto?

Reply:
I don't know about therapist, maybe yes.

Reply:
I guess it means we need to install therapist in order to have the commit hook? Why not simply configure git to look at the githooks directory?